crates/pallet-erc20-support/src/migrations/v1.rs

```diff
@@ -0,0 +1,20 @@
+//! Migration to Version 1.
+
+use frame_support::{sp_tracing::info, traits::Get, weights::Weight};
+
+use crate::BalanceOf;
+use crate::{Approvals, Config};
+
+/// Migrate from version 0 to 1.
+pub fn migrate<T: Config<I>, I: 'static>() -> Weight {
+    info!("Running migration to v1");
+
+    let mut weight: Weight = T::DbWeight::get().reads(0);
```

Comment:
`reads(0)`?

Reply:
Yep, as we should start from zero.

Reply:
This could be better expressed with something like `::new()` or `::default()` or `::zero()` (not sure which ones are available).

Reply:
Actually, just `Wight::new`

Reply:
https://github.com/humanode-network/humanode/pull/809/commits/23c0d22b16e810e4bb13d71a3005000968b8df8d